serialization/testing_utilities.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto2";
+
+package principia.serialization;
+
+// An auxiliary message used to represent tabulated data.  Useful for tests that
+// verify precomputed results.  The arguments and values are to be interpreted
+// by the test.
```

Comment:
.proto files are cheap, and we are using this for only one table; just give it fields m, u, s, c, d (or more verbose names as appropriate).

Reply:
Proto are not so cheap when it comes to compilation time.  Furthermore, this proto will be used in the next PR to test the integrals.  I'm keeping it as is for now, we can revisit when we have more use cases.  (It's not like it's really error-prone: if you swap `m` and `u` for instance, the test is going to fail all over the place.)